doc/stats.rst

```diff
@@ -0,0 +1,106 @@
+`Pairtools stats` as a source of quality control metrics
+===========================================
+
+Overview
+--------
+
+`pairtools stats` produces a human-readable nested dictionary of statistics stored in
+a YAML file or a tab-separated text table (specified through the parameters).
+
+- **Summary statistics** include:
+    - number of pairs (total, unmapped, single-side mapped, etc.),
+    - total number of different pair types (UU, NN, NU, and others, see ` Pair types in pairtools docs <https://pairtools.readthedocs.io/en/latest/formats.html#pair-types>`_),
+    - fraction of duplicates
+    - fraction of cis interactions out of total
+    - fraction of contacts between certain chromosomes
+
+Summary statistics can inform you about the quality of the data.
+For example, more trans interactions can be a sign of Hi-C of protocol problems and lower signal-to-noise ratio.
+Substantial mapping to mitochondrial chromosome (chrM) might be a sign of random ligation.
+
+- **P(s), or scaling.**  The dependence of contact frequency on the genomic
+distance referred to as the P(s) curve or scaling, which is a rich source of information of information and quality of 3C+ experiments.
+The shape of P(s) is often used to characterize mechanisms of genome folding and reveal issues with QC.
+
+Interactive visualization of stats with MultiQC
```

Comment:
Maybe we have to explicitly mention that it's not (yet) available in the main MultiQC repository, so has to be installed from our fork?